RoaringBitmap/src/main/java/org/roaringbitmap/art/Node48.java

```diff
@@ -63,15 +66,15 @@ public void replaceNode(int pos, Node freshOne) {
   @Override
   public int getMinPos() {
     int pos = 0;
-    for (int i = 0; i < 32; i++) {
+    for (int i = 0; i < LONGS_USED; i++) {
       long longv = childIndex[i];
       if (longv == INIT_LONG_VALUE) {
         //skip over empty bytes
-        pos += 8;
+        pos += BYTES_PER_LONG;
         continue;
       } else {
-        for (int j = 0; j <= 7; j++) {
-          byte v = (byte) (longv >>> ((7 - j) * 8));
+        for (int j = 0; j < BYTES_PER_LONG; j++) {
+          byte v = (byte) (longv >>> ((7 - j) <<3));
```

Comment:
Shouldn't we replace the 7 here with `BYTES_PER_LONG - 1` as well, to be consistent? Otherwise one needs to know the relationship between 7 and `BYTES_PER_LONG` when reading the code.

Reply:
Yes fair point, a const that is already bytes_per_long - 1 would be cleaner.  But will do. It implies the shift values should be const as well.